packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_masternode_update.test.ts

```diff
@@ -0,0 +1,979 @@
+import { JsonRpcClient } from '@defichain/jellyfish-api-jsonrpc'
+import { RegTest, RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { OP_CODES, Script, TransactionSegWit, UpdateMasternode, Vin, Vout } from '@defichain/jellyfish-transaction'
+import { fromAddress, P2PKH, P2SH, P2WPKH } from '@defichain/jellyfish-address'
+import { DeFiDRpcError, MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { getProviders, MockProviders } from '../provider.mock'
+import { P2WPKHTransactionBuilder, Prevout } from '../../src'
+import { fundEllipticPair, sendTransaction } from '../test.utils'
+import { Bech32 } from '@defichain/jellyfish-crypto'
+import { BigNumber } from '@defichain/jellyfish-json'
+import { Testing, TestingGroup } from '@defichain/jellyfish-testing'
+
+describe('UpdateMasternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  let providers: MockProviders
+  let builder: P2WPKHTransactionBuilder
+  let jsonRpc: JsonRpcClient
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForWalletBalanceGTE(1000)
+
+    jsonRpc = new JsonRpcClient(await container.getCachedRpcUrl())
+    providers = await getProviders(container)
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  beforeEach(async () => {
+    await providers.randomizeEllipticPair()
+    builder = new P2WPKHTransactionBuilder(providers.fee, providers.prevout, providers.elliptic, RegTest)
+    await fundEllipticPair(container, providers.ellipticPair, 10)
+    await providers.setupMocks()
+
+    // enable updating
+    await jsonRpc.masternode.setGov({
+      ATTRIBUTES: {
+        'v0/params/feature/mn-setowneraddress': 'true',
+        'v0/params/feature/mn-setoperatoraddress': 'true',
+        'v0/params/feature/mn-setrewardaddress': 'true'
+      }
+    })
+    await container.generate(1)
+
+    const balance = await container.call('getbalance')
+    expect(balance >= 2).toBeTruthy()
```

Comment:
any reason why we are asserting the balance to be `>=2`?

Reply:
Oh my bad, I don't need this.
Will remove it 😅